packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -431,6 +428,32 @@ function unwindWork(
       const effectTag = workInProgress.effectTag;
       if (effectTag & ShouldCapture) {
         workInProgress.effectTag = (effectTag & ~ShouldCapture) | DidCapture;
```

Comment:
What's the difference between DidCapture effect and alreadyCaptured?

Reply:
`DidCapture` effect isn't being used for SuspenseComponent anymore, just forgot to remove it. (It's still used by error boundaries).

The motivation for storing this on the state object is to unify the concurrent and non-concurrent modes as much as possible. The begin phase no longer distinguishes between them at all, unlike before where a non-null `updateQueue` was a hack to indicate non-current mode.